app/controllers/api/functions.php

```diff
@@ -158,10 +156,11 @@
     ->inject('dbForProject')
     ->inject('project')
     ->inject('user')
-    ->inject('events')
+    ->inject('queueForEvents')
+    ->inject('queueForBuilds')
```

Comment:
I don't see these resources created in the diff of the `init.php` file. Can you check if these have been added?

Reply:
they are